server/src/integrations/bacen/integration.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+const scheduler = require('node-cron');
+const currency = require('../../db/model/currencyModel');
+
+const getCurrencies = async function integration(currencies, socket) {
+
+  scheduler.schedule('* * * * *', async function () {
+
+    const response = await fetch(`https://economia.awesomeapi.com.br/last/${currencies}`);
```

Comment:
É uma boa prática mover o domínio `https://economia.awesomeapi.com.br` para uma variável de ambiente. Em cenário reais você pode ter um domínio para cada ambiente: `test` `stage` e `prod`.

```suggestion
    const { ECONOMY_ENDPOINT } = process.env;
    const response = await fetch(`${ECONOMY_ENDPOINT}/last/${currencies}`);
```